Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/InputValue.cs

```diff
@@ -35,6 +35,43 @@ public object Get()
         }
 
         ////TODO: add automatic conversions
+        /// <summary>
+        /// Read the value of the action.
+        /// </summary>
+        /// <returns>The current value from the action cast to the specified type.</returns>
+        /// <typeparam name="TValue">Type of value to read. This must correspond to the
+        /// expected by either <see cref="control"/> or, if it is a composite, by the
+        /// <see cref="InputBindingComposite"/> in use.
+        /// The type depends on what type of controls the action is bound to. Common types are float and Vector2</typeparam>
+        /// <exception cref="InvalidOperationException">The given type <typeparamref name="TValue"/>
+        /// does not match the value type expected by the control or binding composite.</exception>
+        /// <remarks>
+        /// The following example shows how to read a value from a <see cref="PlayerInput"/> message.
+        ///
+        /// <example>
+        /// <code>
+        /// [RequireComponent(typeof(PlayerInput))]
+        /// public class MyPlayerLogic : MonoBehaviour
+        /// {
+        ///     private Vector2 m_Move;
+        ///
+        ///     // 'Move' input action has been triggered.
+        ///     public void OnMove(InputValue value)
+        ///     {
+        ///         // Read value from control. The type depends on what type of controls the action is bound to.
+        ///         m_Move = value.Get&lt;Vector2&gt;();
+        ///     }
+        ///
+        ///     public void OnUpdate()
+        ///     {
+        ///         // Update transform from m_Move
+        ///     }
+        /// }
+        /// </code>
+        /// </example>
+        /// The given InputValue is only valid for the duration of the callback. Storing the InputValue references somewhere and calling Get&lt;T&gt;() later does not work correctly.
```

Comment:
I think the standard suggests using "input value" instead of the formal name. If formal, use <c></c> fir fixed formatting I believe.

Reply:
I copied this from elsewhere in the docs so I'm tempted to leave this.

Reply:
I've changed to `<c>InputValue</c>`